### DISCUSSION TITLE
How to use Drawio with tooltip enabled in mkdocs material

### DISCUSSION THREAD

**Discussion by Sahayavinodh** [squidfunk/mkdocs-material#7184] [Q&A] [answered]
How to use Drawio with tooltip enabled in mkdocs material? I saved the Drawio image  as png, svg and html and tried using in mkdocs material theme but I am not seeing the text which is entered using tooltip. The reason I didn't use mkdocs tooltip. The image will have multiple logos and when I hoover over each logo it will display a text. Can you please help on this.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @Sahayavinodh,
please provide a more specific example, best a whole [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#minimal-reproduction) we could run to test things out on our end.

Looking at the readme of https://pypi.org/project/mkdocs-drawio/ it seems to me that the drawio diagram is inserted via some kind of JavaScript into the page. It might be generated in a shadow root / iframe, and therefore not be visible for the Tooltip logic in `mkdocs-material` as this requires for the HTML elements to be in the main DOM. Or perhaps you're not using that plugin, and instead embedding the diagram yourself inside of some iframe, then the same assumption applies.

Without further details, I assume this requires a lot of customization to get to work.

**Comment by Sahayavinodh** [comment]
Thanks,

After adding the plugins it worked. Thank you for your input.

Thanks,
Sahaya Vinodh

**Reply by kamilkrzyskow** [reply]
OK, good to know, if you don't want to elaborate on the technicalities why it started working, what is the difference between what the plugin is doing and what you did, then I will mark comment as the answer for now :v:
